src/index.tsx

```diff
@@ -671,8 +671,26 @@ class App extends React.Component<{}, AppState> {
                     if (!e.shiftKey) {
                       clearSelection(elements);
                     }
-                    // No matter what, we select it
-                    hitElement.isSelected = true;
```

Comment:
You've moved this out by mistake I think, but it doesn't matter, and I'll probably refactor this part of the code later, anyway, so...